terracumber_config/tf_files/SUSEManager-4.2-PRV.tf

```diff
@@ -189,8 +189,20 @@ module "cucumber_testsuite" {
     kvm-host = {
       image = "sles15sp3o"
       additional_grains = {
-        hvm_disk_image = "http://minima-mirror.mgr.prv.suse.net/distribution/leap/15.3/appliances/openSUSE-Leap-15.3-JeOS.x86_64-OpenStack-Cloud.qcow2"
-        hvm_disk_image_hash = "http://minima-mirror.mgr.prv.suse.net/distribution/leap/15.3/appliances/openSUSE-Leap-15.3-JeOS.x86_64-OpenStack-Cloud.qcow2.sha256"
+        hvm_disk_image = {
+          leap = {
+            name = "leap"
```

Comment:
You probably can get rid of the `name` values in this PR too

Reply:
Fixed in sync with the sumaform changes. Thank you.